applications/validate_camera_fov.py

```diff
@@ -47,5 +68,7 @@
 
     # Now plot the camera as well
     plt = camera.plotPixelLayout()
-    plt.savefig('pixelLayout-LST.pdf', format='pdf', bbox_inches='tight')
+    cameraPlotFile = 'pixelLayout-{}.pdf'.format(telModel.telescopeType)
+    plt.savefig(cameraPlotFile, bbox_inches='tight')
```

Comment:
Note for the future: We need to figure out a system to handle the application plots.

Reply:
Yes, we should find a solution. I don't know if the DB will provide a solution to this, it probably needs to be produced locally and optionally be pushed to the DB. Requires a bit more thinking I guess...